global.json

```diff
@@ -5,7 +5,12 @@
     "rollForward": "major"
   },
   "tools": {
-    "dotnet": "8.0.101"
+    "dotnet": "8.0.101",
+    "runtimes" : {
+      "dotnet" : [
+        "6.0.27"
```

Comment:
Not sure what's better, but we could also use property syntax here and define the version in our version.props file. 
https://github.com/dotnet/arcade/blob/130b08562936277405284145c1befc87bf1e83cb/Documentation/ArcadeSdk.md?plain=1#L411

Reply:
No preference but one might integrate better with an update bot.